### PR TITLE
Style login  & register page

### PR DESCRIPTION
**Pull request by kathinka**
- added global styles 
- styled login and register page
- added logo and globe
- fresh build of frontend
- added favicon, imported font and updated page meta desc.

### CODE REVIEW

File: frontend/src/Globalstyles.jsx [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
-    --warning-color: #ffc107;
-    --info-color: #17a2b8;
-    --light-color: #f8f9fa;
-    --dark-color: #343a40;
+    --ego-white: #fff;
+    --ego-green: #687943;
+    --ego-lgt-green: #CCE0A1;
+    --ego-beige: #DCDED0;
+    --ego-purple: #C590FB;
+    --ego-error: #794343;
```
Comment (El1an3) [inline frontend/src/Globalstyles.jsx:10]: Yeah to global styles. 
Did you add new colors? This one is not in the design.

File: frontend/src/assets/logo.svg
Comment (El1an3) [inline frontend/src/assets/logo.svg:1]: There's a different logo in the design

**Review by El1an3 — APPROVED** [review]
Looks good, except some minor styling things that can easily be fixed. 
Like logo, image being a bit big on mobile and form elements that go to the right when in between mobile and desktop size. 
And a different color for errors than in the design.

File: frontend/src/Globalstyles.jsx (hunk 1 above)
Comment (Yifan-858) [reply frontend/src/Globalstyles.jsx:10]: Great to the global variables!
Yeah, I also noticed. the error box color is also --ego-purple: #C590FB; :)

**Review by Yifan-858 — APPROVED** [review]
Thank you Kathinka! Looks good for me! There are some minor things, like the position of the globe, but I'm sure they will be fixed later when we put more focus on the resposiveness :) Good to go!

File: frontend/dist/assets/Globe-49d51795.svg
Comment (Yifan-858) [inline frontend/dist/assets/Globe-49d51795.svg:1]: hmmm.. this image is somehow cropped. The purple lines are not fully displayed. Worth looking into? It seems to be the reason that the image is cut-off on the login page.

Comment (kathinka) [reply frontend/dist/assets/Globe-49d51795.svg:1]: hmm.. Its just the svg i exported from figma, but maybe that one was in a clipmask. i'll check and update it :)

File: frontend/src/assets/logo.svg
Comment (kathinka) [reply frontend/src/assets/logo.svg:1]: I just used the logo i found in the sketches -will have another look  and update

File: frontend/src/Globalstyles.jsx (hunk 1 above)
Comment (kathinka) [reply frontend/src/Globalstyles.jsx:10]: added some who I thought we needed . will fix the error color :)

**Comment by kathinka** [comment]
ok, so now that i had the right figmafile it was easy to update the design. new pr coming up!

**Pull request merged**